MailCore/Models/Draft.swift

```diff
@@ -375,9 +371,22 @@ public extension Draft {
     }
 
     var isCompletelyEmpty: Bool {
```

Comment:
I think we should rename this function as well, it is named `isCompletelyEmpty`. 
But whe check with `isEmpty(removeAllElements: false)`.

Maybe a `isEmptyOfUserChanges` ?